float64 x
float64 y
float64 yaw
float64 x_vel
float64 y_vel
float64 yaw_vel